src/app/files/file-explorer.js

```diff
@@ -276,6 +316,217 @@ fileExplorer.prototype.init = function () {
   return this.container
 }
 
+fileExplorer.prototype.publishToGist = function (fileProviderName) {
```

Comment:
I think these functions don't not have any param anymore?

Reply:
oh, forgot to remove this